website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -1029,21 +1374,63 @@ $ curl \
 }
 ```
 
-## Add tags to a workspace
+_List key-value tags bound directly to the workspace:_
 
-This endpoints adds one or more tags to the workspace. The workspace must already exist, and any
-tag element that supplies an `id` attribute must exist. If the `name` attribute is used, and no
-matching organization tag exists with such name, a new one will be created.
+```json
+{
+  "data": [
+    {
+      "id": "tb-RKs9JSC2lInns32K",
+      "type": "tag-bindings",
+      "attributes": {
+        "key": "ws1-key",
+        "value": "ws1-value"
+      }
+    }
+  ]
+}
+```
 
-Tags are subject to the following rules:
+_List all key-value tags bound to the workspace, including those inherited from the parent project:_
 
-- Workspace tags or tags must be one or more characters, have a 255 character limit, and can include letters, numbers, colons, hyphens, and underscores.
-- You can create tags for a workspace using the user interface or the API. After you create a tag, you can assign it to other workspaces in the same organization.
-- You cannot create tags for a workspace using the CLI.
-- You cannot set tags at the project level, so there is no tag inheritance from projects to workspaces.
+```json
+{
+  "data": [
+    {
+      "type": "effective-tag-bindings",
+      "attributes": {
+        "key": "ws1-key",
+        "value": "ws1-value"
+      }
```

Comment:
I think we're going to need to return some kind of ID here or we will [violate the spec](https://jsonapi.org/format/#document-resource-objects)

Reply:
Any suggestions? I think we decided not to give these external IDs